Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -89,69 +83,47 @@ class TabCell: UICollectionViewCell {
         self.titleText.isUserInteractionEnabled = false
         self.titleText.numberOfLines = 1
         self.titleText.font = DynamicFontHelper.defaultHelper.DefaultSmallFontBold
+        self.titleText.textColor = LightTabCellUX.TabTitleTextColor
+        self.titleText.backgroundColor = .clear
 
         self.closeButton = UIButton()
-        self.closeButton.setImage(#imageLiteral(resourceName: "tab_close").template, for: [])
+        self.closeButton.setImage(#imageLiteral(resourceName: "tab_close"), for: [])
```

Comment:
Why removing `template`? Just not needed in this instance?

Reply:
No need in this instance, since `closeButton` is not a `system` typed `UIButton`